make.py

```diff
@@ -9,7 +9,11 @@
 def output_on_failure(cmd, cwd):
     print(f'$ cd {cwd} && {cmd}')
     ret = subprocess.run(
-        cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, cwd=cwd,
+        cmd,
+        cwd=cwd,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT,
+        universal_newlines=True,
```

Comment:
`universal_newlines` has the same problem as `.decode()` or `encoding=...` or `text=True` in that it hides the actual error on a decoding issue